torch/export/_trace.py

```diff
@@ -1062,7 +1062,7 @@ def forward(self, *args, **kwargs):
                 new_fake_constant_attrs,
                 map_fake_to_real,
             ):
-                ep_non_strict = _export_non_strict(
+                aten_ep = _export_to_aten_ir(
```

Comment:
same with these -- it's not really an ExportedProgram yet so I don't think we should name it "ep". Maybe like, "exported_artifact"